### PR TITLE
feat(azure): be able to define a required reviewer for PRs

### PR DESCRIPTION
**Pull request by sch1ldkr0ete**
## Changes

When a reviewer for an Azure DevOps PR is prefixed with `required:` the reviewer will be marked as a required reviewer in the Azure DevOps PR.

## Context

Please have a look at the discussion [#17659](https://github.com/renovatebot/renovate/discussions/17659).

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [x] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/modules/platform/azure/index.ts [hunk 1]
```diff
@@ -778,6 +778,7 @@ async function getUserIds(users: string[]): Promise<User[]> {
   const azureApiCore = await azureApi.coreApi();
   const repos = await azureApiGit.getRepositories();
   const repo = repos.filter((c) => c.id === config.repoId)[0];
+  const reqReviewerIndicator = 'required:';
```
Comment (JamieMagee) [inline lib/modules/platform/azure/index.ts:781]: tiny nit

```suggestion
  const requiredReviewerPrefix = 'required:';
```

**Review by JamieMagee — CHANGES_REQUESTED** [review]
Overall looks good. Just some lint and test fixes before I can sign off and merge.

File: lib/modules/platform/azure/index.ts (hunk 1 above)
Comment (sch1ldkr0ete) [reply lib/modules/platform/azure/index.ts]: Thanks! I used your suggestion.

**Comment by sch1ldkr0ete** [comment]
> Overall looks good. Just some lint and test fixes before I can sign off and merge.



> Overall looks good. Just some lint and test fixes before I can sign off and merge.

Done:)

**Review by JamieMagee — DISMISSED** [review]

**Comment by sch1ldkr0ete** [comment]
Just fixed a remaining linting issue in the docs. Hope that was okay.

Should I add some tests for keeping the code coverage percentage?

**Comment by viceice** [comment]
> Just fixed a remaining linting issue in the docs. Hope that was okay.
> 
> Should I add some tests for keeping the code coverage percentage?

yes, needs to stay on 100%

**Comment by sch1ldkr0ete** [comment]
I marked one of the azure test reviewers as required. The coverage should be back to 100%:
image

**Review by JamieMagee — APPROVED** [review]

**Pull request merged**